cli/lib/kontena/cli/stacks/install_command.rb

```diff
@@ -55,7 +56,9 @@ def install_dependencies
 
         cmd << dependency['stack']
         Kontena.run!(cmd)
+        set_env_variables(stack_name, current_grid)
```

Comment:
While this probably fixes the issue, it might be clearer to just change the `#execute` method to look like:
```ruby
    def execute
      install_dependencies unless skip_dependencies?

      set_env_variables(stack_name, current_grid)
      stack # runs validations
```

But note that this is assuming that:
* `loader.dependencies` doesn't touch the `self.stack` -> `@stack ||= self.reader.execute`
* `install_dependencies` doesn't need the `ENV['STACK']` to be set...?